RecoTracker/TkSeedGenerator/plugins/BuildFile.xml

```diff
@@ -31,7 +31,7 @@
 <use name="MagneticField/UniformEngine"/>
 <use name="RecoTracker/MeasurementDet"/>
 <use name="RecoTracker/Record"/>
-<use name="RecoTracker/SpecialSeedGenerators"/>
+<use name="RecoTracker/SpecialSeedGenerators" source_only="1"/>
```

Comment:
(Trying to assimilate the usage od the `source_only` flag in the BuildFile's, I looked around for "RecoTracker/SpecialSeedGenerators"): shouldn't you have one such declaration also in RecoTracker/​ConversionSeedGenerators/​plugins/​BuildFile.xml?

Reply:
What does `source_only="1"` do ?

Reply:
From https://github.com/cms-sw/cmssw/pull/31459#issuecomment-692737732 I understood that the `source_only` flag makes sense if you depend on a CMSSW package that is source only, but it still has a BuildFile which is useful to inherit dependencies from.

However, I fix here some warnings that arise when a source-only package has no BuildFile. So for this cases, the `use name` line with `source_only` actually does nothing currently. But if in the future the those source-only package will get a Build file, we can profit from it. So adding the `source_only` flag appears to be more future-proof than dropping the line.

Still, I also think that this argument is not *that* important that we need to preemptively add lines with `source_only` if there is no warning or other inconsistency.